src/currencies/getDeviceUnit.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import invariant from "invariant";
+import type { Account, Unit } from "../types";
+
+/**
+ * getDeviceUnit returns the unit that appears on the device, based on the
+ * user preferred unit for the account.
+ *
+ * Normally, it is the same as account.unit, except for testnet accounts,
+ * where the unit on device is just "TEST".
+ */
+export function getDeviceUnit(account: Account): Unit {
```

Comment:
I suggest a small simplification, we need need to have `deviceUnits?: Unit[]` but just a `deviceTicker?: string`

I think especially we can:
- in CryptoCurrency type a `deviceTicker?: string` 
- define for bitcoin_testnet `deviceTicker: "TEST"`
- in the deviceAction, you can `account.currency.deviceTicker || account.unit`